modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserAccountResourceTest.java

```diff
@@ -2234,6 +2245,19 @@ private void _testGetUserAccountWithMoreExternalReferenceCodes()
 	private void _testGetUserAccountWithNestedFields() throws Exception {
 		User user = UserTestUtil.addUser();
 
+		DepotEntry depotEntry = _depotEntryLocalService.addDepotEntry(
+			HashMapBuilder.put(
+				LocaleUtil.getDefault(), RandomTestUtil.randomString()
+			).build(),
+			new HashMap<>(),
+			ServiceContextTestUtil.getServiceContext(
+				GroupLocalServiceUtil.getGroup(
+					TestPropsValues.getCompanyId(), GroupConstants.GUEST),
+				user.getUserId()));
```

Comment:
what do you need here? Why cannot we use the usual TestPropertyValue.getCompanyId and TestPropertyValue.getGroupId?